exonum/src/node/events.rs

```diff
@@ -49,16 +52,40 @@ impl NodeHandler {
     fn handle_api_event(&mut self, event: ExternalMessage) {
         match event {
             ExternalMessage::Transaction(tx) => {
+                if !self.is_enabled {
+                    info!("Ignoring the transaction {:?}", tx);
```

Comment:
Hmm, I'm not sure, maybe we should also print *why* the transaction is ignored. It can be confusing otherwise.
But consider this as a nitpick